packages/ts-morph/src/compiler/types/Type.ts

```diff
@@ -233,9 +233,19 @@ export class Type<TType extends ts.Type = ts.Type> {
     }
 
     /**
-     * Gets the target type of a type reference if it exists.
+     * Gets the reference type if this type is an instantiated type.
      *
-     * For example, given `Promise<string>` this will return `Promise<T>`.
+     * Specifically:
+     *
+     * - If this type is an instantiated type then returns the reference type.
+     * - If this type is a reference type then returns itself (same reference).
+     * - Otherwise returns `undefined`.
```

Comment:
Maybe: "Otherwise returns `undefined` when not a type reference."

Reply:
Technically its "when not a type reference OR a generic type". I feel it should be either totally explicit or implicit, as sayig in-between might confuse the message?

Reply:
Yes, true!